RomanNumeralsConverter.Tests/ValidationRulesTests/InputGivenValidatorTests.cs

```diff
@@ -0,0 +1,37 @@
+using FluentAssertions;
+using NUnit.Framework;
+using RomanNumeralsConverter.ValidationRules;
+
+namespace RomanNumeralsConverter.Tests.ValidationRulesTests
+{
+  internal class InputGivenValidatorTests
+  {
+    private InputGivenValidator _subject;
+
+    [SetUp]
+    public void SetUp()
+    {
+      _subject = new InputGivenValidator();
+    }
+
+    [TestCase(null)]
+    public void Should_ReturnNullInputValidationError_When_InputIsNull(string romanNumeral)
```

Comment:
There is no such a thing like input. Again it is very low level name you can be more concrete here